cmake/modules/Findlibcap.cmake

```diff
@@ -0,0 +1,33 @@
+# - Find libcap
+# Sets the following:
+#
+# LIBCAP_INCLUDE_DIR
+# LIBCAP_LIBRARIES
+# LIBCAP_VERSION
+# LIBCAP_FOUND
+
+find_package(PkgConfig QUIET REQUIRED)
+pkg_search_module(PC_libcap libcap)
+
+find_path(LIBCAP_INCLUDE_DIR
+        NAMES sys/capability.h
+        PATHS ${PC_libcap_INCLUDE_DIRS})
+
+find_library(LIBCAP_LIBRARIES
+        NAMES libcap.so
```

Comment:
if we don't really care about whether to use shared library or static library, probably could just put the library name instead of the file name? like
```cmake
find_library(LIBCAP_LIBRARIES
  NAMES cap
  PATHS ${PC_libcap_LIBRARY_DIRS})
```

also, i'd suggest use two spaces for indent.